tests/integration/workflow/test_model.py

```diff
@@ -82,13 +80,6 @@ def test__load_all() -> None:
     assert model.load_all(tags={"does_not_exist"}) == []
 
 
-def test__load__mismatching_workflows() -> None:
```

Comment:
Alternative is register another generic workflow to keep validations of this kind.

Reply:
If we merge with this removed, we should track follow ups to re-add / re-enable tests in a linear issue.

Reply:
Added back